src/DGtal/shapes/doc/moduleSurfaceMesh.dox

```diff
@@ -0,0 +1,237 @@
+/**
+* @file moduleSurfaceMesh.dox
+*
+* @author Jacques-Olivier Lachaud (\c jacques-olivier.lachaud@univ-savoie.fr )
+* Laboratory of Mathematics (CNRS, UMR 5807), University of Savoie, France
+*
+* @date 2017/05/7
+*
+* Documentation file for module moduleSurfaceMesh
+*
+* This file is part of the DGtal library.
+*/
+
+/* 
+* Useful to avoid writing DGtal:: in front of every class.
+* Do not forget to add an entry in src/DGtal/base/Config.h.in !
+*/
+namespace DGtal {
+//----------------------------------------
+/*!
+@page moduleSurfaceMesh Surface mesh data structure for representing manifold or non-manifold polygonal surfaces in R3
+@author Jacques-Olivier Lachaud
+
+@since 1.1
+
+Part of \ref packageShapes
+  
+This part of the manual describes how to represent and manipulate
+generic polygonal surfaces embedded in \f$ \mathbb{R}^3 \f$. The class
```

Comment:
If we have a quick "Mesh types / variations" module page (see above), add ref to it here

Reply:
done